src/main/kotlin/com/openlattice/conductor/rpc/ConductorConfiguration.kt

```diff
@@ -0,0 +1,36 @@
+package com.openlattice.conductor.rpc
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.kryptnostic.rhizome.configuration.Configuration
+import com.kryptnostic.rhizome.configuration.ConfigurationKey
+import com.kryptnostic.rhizome.configuration.SimpleConfigurationKey
+import com.kryptnostic.rhizome.configuration.annotation.ReloadableConfiguration
+
+
+private val REPORT_EMAIL_ADDRESS_FIELD = "reportEmailAddress"
+private val SEARCH_CONFIGURATION_FIELD = "searchConfiguration"
+private val key = SimpleConfigurationKey("conductor.yaml")
+
+/**
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+@ReloadableConfiguration(uri = "conductor.yaml")
+class ConductorConfiguration(
+        @JsonProperty("reportEmailAddress") val reportEmailAddress: String,
+        @JsonProperty("searchConfiguration") val searchConfiguration: SearchConfiguration,
+        @JsonProperty("bootstrap-connection") val connection: String
+) : Configuration {
+    @JsonIgnore
+    override fun getKey(): ConfigurationKey {
+        return key
+    }
+
+    companion object {
+        @JvmStatic
+        fun key(): ConfigurationKey {
+            return key
```

Comment:
Won't this go into an infinite loop?

Reply:
I just remembering having this issue in IndexerConfiguration and that's why I renamed it there to configKey

Reply:
ah yeah I remember this fun kotlin experience

Reply:
I am not having this problem? I wonder if this has been fixed in the latest version of kotlin.